tests/trading/test_shareToken.py

```diff
@@ -114,7 +114,7 @@ def test_approve(contractsFixture, market):
             "to": bytesToHexString(tester.a1),
             "universe": market.getUniverse(),
             "value": 7
-        },
+        }        
```

Comment:
Style: Remove trailing whitespace.

Reply:
Also style, though not one I think we have discussed. I prefer trailing commas as it makes re-arranging things easier.